src/angular/schematics/ng-add/test-cases/merge/pagination_input.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  SbbPageChangeEvent,
+  SbbPaginationModule,
+  SbbPaginatorComponent
+} from '@sbb-esta/angular-public';
+import { Component, NgModule } from '@angular/core';
+
+@Component({
+  template: `
+    <sbb-pagination (pageChange)="changePage($event)" length="5"></sbb-pagination>`
```

Comment:
Please add two or three additional examples to make sure everything works as expected.
See examples at https://code.sbb.ch/plugins/servlet/search?q=%22sbb-pagination%22%20ext%3Ahtml